docs/evaluation/simulate.rst

```diff
@@ -186,15 +282,19 @@ numbers as background value and then performing a Poisson draw on them
 again gives a higher level of noise than a real observation would
 have. To avoid this problem, a background model may be used instead::
 
-  >>> from sherpa.models.basic import Const1D
-  >>> bkgmdl = Const1D('bmdl')
-  >>> bkgmdl.c0 = 2
-  >>> fake_pha(data, mdl, add_bkgs=True, bkg_models={'1': bkgmdl})
+   >>> from sherpa.models.basic import Const1D
+   >>> bkgmdl = Const1D('bmdl')
+   >>> bkgmdl.c0 = 2
+   >>> fake_pha(data, mdl, add_bkgs=True, bkg_models={'1': bkgmdl})  # doctest: +SKIP
 
 The keys in the `bkg_models` dictionary are the identifiers of the
 backgrounds. Above, we loaded a background with the default identifier
 (which is ``1``).
 
+.. todo::
+   Looks like we need `instmdl` here, but does this mean that
+   bkg_models needs to be sent `resp(bkgmdl)` ?
+
```

Comment:
Good question. We should know the answer to that, since we wrote the code...

And the code says that:
```Python
    if bkg_models is not None:
        warn("bkg_models is no-longer used, as the model "
             "must contain any background component",
             category=DeprecationWarning)
```

So, the example above must be rewritten to make `bkgmdl` a part of the model expression like `combined_mdl = resp(srcmdl + bkgmdl)` if the `bkgmdl` represents an astrophysical background. Of course, it could also be an instrumental background (e.g. high-energy proton flux) that is independent of the response, in which case `combined_mdl = resp(srcmdl) + bkgmdl` might be more appropriate. I suggest mentioning both possibilites without going into the reasons why one might want to use the one or the other. Suggested text:

If a background model represents and astrophysical component, e.g. the emission of a nebula behind the source we want to simulate, then the background model needs to apply the instrumental response in the same way as the source spectrum: `combined_mdl = resp(srcmdl + bkgmdl)`. On the other hand, the background model might describe electronic noise on a CCD, high-energy protons or some other instrumental effect that is independent of the mirror effective area. In this more complicated case, the instrumental ARF and RMF might not be appropriate; instead the user might set up a different response for the background.